diff --git a/modules/CMakeLists.txt b/modules/CMakeLists.txt
index 46c9d5e16..f8256593c 100644
--- a/modules/CMakeLists.txt
+++ b/modules/CMakeLists.txt
@@ -35,8 +35,8 @@ CU_GET_GLOBAL("AC_ADD_SCRIPTS_INCLUDE")
 set("AC_SCRIPTS_INCLUDES" "")
 set("AC_MODULE_LIST" "")
 set("AC_SCRIPTS_LIST" "")
-set(MOD_ELUNA_FOUND 0)
-set(MOD_ELUNA_PATH "")
+set(MOD_ECLIPSE_FOUND 0)
+set(MOD_ECLIPSE_PATH "")
 
 foreach(include ${AC_ADD_SCRIPTS_INCLUDE})
   set("AC_SCRIPTS_INCLUDES" "#include \"${include}\"\n${AC_SCRIPTS_INCLUDES}")
@@ -51,9 +51,9 @@ foreach(scriptName ${AC_ADD_SCRIPTS_LIST})
 endforeach()
 
 function(ConfigureElunaModule moduleName)
-  set(MOD_ELUNA_FOUND 1 PARENT_SCOPE)
+  set(MOD_ECLIPSE_FOUND 1 PARENT_SCOPE)
   GetPathToModuleSource(${SOURCE_MODULE} MODULE_SOURCE_PATH)
-  set(MOD_ELUNA_PATH ${MODULE_SOURCE_PATH} PARENT_SCOPE)
+  set(MOD_ECLIPSE_PATH ${MODULE_SOURCE_PATH} PARENT_SCOPE)
 
   # Define eluna compile options
   target_compile_options(game-interface
@@ -77,7 +77,7 @@ foreach(SOURCE_MODULE ${MODULES_MODULE_LIST})
   endif()
 
   # Use only static for mod-eluna
-  if (SOURCE_MODULE MATCHES "mod-eluna")
+  if (SOURCE_MODULE MATCHES "mod-eclipse")
     ConfigureElunaModule(${SOURCE_MODULE})
   endif()
 
@@ -267,9 +267,9 @@ ConfigureScriptLoader("static" SCRIPT_MODULE_PRIVATE_SCRIPTLOADER OFF ${STATIC_S
 
 list(REMOVE_DUPLICATES SCRIPT_MODULE_PRIVATE_SCRIPTLOADER)
 
-if (MOD_ELUNA_FOUND)
-  list(REMOVE_ITEM PRIVATE_SOURCES_MODULES ${MOD_ELUNA_PATH}/lualib/lua.c)
-  list(REMOVE_ITEM PRIVATE_SOURCES_MODULES ${MOD_ELUNA_PATH}/lualib/luac.c)
+if (MOD_ECLIPSE_FOUND)
+  list(REMOVE_ITEM PRIVATE_SOURCES_MODULES ${MOD_ECLIPSE_PATH}/lualib/lua.c)
+  list(REMOVE_ITEM PRIVATE_SOURCES_MODULES ${MOD_ECLIPSE_PATH}/lualib/luac.c)
 endif()
 
 add_library(modules STATIC
@@ -277,8 +277,8 @@ add_library(modules STATIC
     ${SCRIPT_MODULE_PRIVATE_SCRIPTLOADER}
     ${PRIVATE_SOURCES_MODULES})
 
-if (MOD_ELUNA_FOUND)
-    target_link_libraries(modules PUBLIC lualib)
+if (MOD_ECLIPSE_FOUND)
+    target_link_libraries(modules PUBLIC lualib sol2::sol2)
 endif()
 
 target_link_libraries(modules
@@ -357,7 +357,7 @@ target_compile_options(modules
   INTERFACE
     -DCONFIG_FILE_LIST=$<1:"${CONFIG_LIST}">)
 
-if (MOD_ELUNA_FOUND)
+if (MOD_ECLIPSE_FOUND)
   if (APPLE)
     target_compile_definitions(modules
       PUBLIC
@@ -376,10 +376,10 @@ if (MOD_ELUNA_FOUND)
     add_custom_command(TARGET modules
     POST_BUILD
     COMMAND ${CMAKE_COMMAND} -E make_directory "${CMAKE_BINARY_DIR}/bin/${MSVC_CONFIGURATION_NAME}lua_scripts/extensions/"
-    COMMAND ${CMAKE_COMMAND} -E copy_directory "${MOD_ELUNA_PATH}/LuaEngine/extensions" "${CMAKE_BINARY_DIR}/bin/${MSVC_CONFIGURATION_NAME}lua_scripts/extensions/")
+    COMMAND ${CMAKE_COMMAND} -E copy_directory "${MOD_ECLIPSE_PATH}/LuaEngine/extensions" "${CMAKE_BINARY_DIR}/bin/${MSVC_CONFIGURATION_NAME}lua_scripts/extensions/")
   endif()
 
-  install(DIRECTORY "${MOD_ELUNA_PATH}/LuaEngine/extensions" DESTINATION "${CMAKE_INSTALL_PREFIX}/bin/lua_scripts/")
+  # install(DIRECTORY "${MOD_ECLIPSE_PATH}/LuaEngine/extensions" DESTINATION "${CMAKE_INSTALL_PREFIX}/bin/lua_scripts/")
 endif()
 
-message("")
+message("")
\ No newline at end of file
